vendor/jasmine-only/jasmine-only.js

```diff
@@ -0,0 +1,98 @@
+/* jasmine-only - 0.1.1
+ * Exclusivity spec helpers for jasmine: `describe.only` and `it.only`
+ * https://github.com/davemo/jasmine-only
+ */
+(function() {
```

Comment:
as a follow-up I would even recommend removing all the compiler shit and just rewriting this in node 4 with classes/arrow functions.